core/entities/content-type.js

```diff
@@ -9,19 +8,10 @@ const listContentTypes = () => contentTypes;
 
 const getContentTypeMeta = contentType => {
   const config = getContentTypeSchema(contentType);
-  const fields = config.fields.map(field => {
```

Comment:
Well, here i tried to remove warnings in console. Being honest I didnt manage to find explaining why we had warnings, but it was because of passing `Text` and `MultiLine` component through `getInitialProps` on server side, now I'm choosing what to render (based on field type) in `edit.jsx`, so warning is fixed